frontend/src/app/shared/components/header/header.component.ts

```diff
@@ -12,9 +18,42 @@ export class HeaderComponent {
 
     user: IUser;
 
-    constructor(private authService: AuthService) {
+    constructor(
+        private authService: AuthService,
+        private modalService: NgbModal,
```

Comment:
we have our shared confirmation window
please use it

Reply:
I used ConfirmationModalComponent:         const modalRef = this.modalService.open(ConfirmationModalComponent, { windowClass: 'log-out-modal' });